package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@acrolinx/sidebar-interface",
   "description": "The interface of the Acrolinx Sidebar",
-  "version": "15.3.1",
+  "version": "15.3.2",
```

Comment:
I don't want to be religious, but I guess it should rather be a new feature version then a patch version

Reply:
Updated to 15.4.0